pkg/tool/helm.go

```diff
@@ -2,82 +2,163 @@ package tool
 
 import (
 	"fmt"
+	"os"
+	"time"
 
-	"github.com/helm/chart-testing/v3/pkg/tool"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/cli/values"
+	"helm.sh/helm/v3/pkg/getter"
+	"k8s.io/helm/pkg/strvals"
 )
 
-// Helm is an interface to the helm binary; it is a thin layer on top of the Helm abstraction offered by chart-testing
-// to silence output being streamed to Stdout.
 type Helm struct {
-	tool.Helm
-	ProcessExecutor
-	extraArgs []string
+	config   *action.Configuration
+	settings *cli.EnvSettings
+	args     map[string]interface{}
 }
 
-func NewHelm(exec ProcessExecutor, extraArgs []string) Helm {
-	return Helm{
-		tool.NewHelm(exec.ProcessExecutor, extraArgs),
-		exec,
-		extraArgs,
+func NewHelm(settings *cli.EnvSettings, args map[string]interface{}) (*Helm, error) {
+	helm := new(Helm)
+	helm.settings = settings
+	helm.args = args
```

Comment:
Not a particular reason. Yes I agree with the proposed coding style, making struct initialization in one assignment instead of scattering multiple lines.